### PR TITLE
remove extra define for __NR_getrandom and add some comments

### PR DESCRIPTION
**Pull request by zzl360**
add __NR_getrandom support for loongarch64

### CODE REVIEW

File: providers/implementations/rands/seeding/rand_unix.c [hunk 1]
```diff
@@ -321,6 +321,8 @@ static ssize_t sysctl_random(char *buf, size_t buflen)
 #     define __NR_getrandom    356
 #    elif defined(__aarch64__)
 #     define __NR_getrandom    278
+#    elif defined(__loongarch__) || defined(__loongarch64)
+#     define __NR_getrandom    278
```
Comment (t8m) [inline providers/implementations/rands/seeding/rand_unix.c:325]: Why do you need this? Isn't the generic fallback below sufficient?

Comment (zzl360) [reply providers/implementations/rands/seeding/rand_unix.c:325]: > Why do you need this? Isn't the generic fallback below sufficient?

I saw there is a same define under __aarch64__ macro. just free to tell me if it is not accpeted.

Comment (t8m) [reply providers/implementations/rands/seeding/rand_unix.c:325]: What about removing the aarch64 case and just add aarch64 and loongarch to the comment in the generic case?

Such as: `#    else /* generic (f.e. aarch64, loongarch, loongarch64) */

Comment (zzl360) [reply providers/implementations/rands/seeding/rand_unix.c]: done

**Review by t8m — APPROVED** [review]

**Review by kroeckx — APPROVED** [review]

**Comment by hlandau** [comment]
Merged to master. Thank you.

**Pull request closed without merging**